src/Api/Projects.php

```diff
@@ -78,6 +78,8 @@ public function all(array $parameters = [])
             'id', 'name', 'path', 'created_at', 'updated_at', 'last_activity_at',
             'repository_size', 'storage_size', 'packages_size', 'wiki_size',
         ];
+        $resolver->setDefined('pagination');
```

Comment:
These parameters are not intentionally not supported. Please remove them. You need to use the result pager class.